portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -75,6 +78,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
         // Get the input from the form.
         String name = getParameter(request, "name", "");
         String text = getParameter(request, "comment", "");
+
+        Entity commentEntity = new Entity("Comment");
+        commentEntity.setProperty("name", name);
+        commentEntity.setProperty("text", text);
```

Comment:
You'll probably want to sort comments chronologically, so a timestamp field is a good idea here.